Content.Client/Gameplay/GameplayStateBase.cs

```diff
@@ -94,7 +93,7 @@ public ClickableEntityComparer(IEntityManager entities)
             public int Compare((EntityUid clicked, int depth, uint renderOrder) x,
                 (EntityUid clicked, int depth, uint renderOrder) y)
```

Comment:
Oher than being negated/reversed, this function should match the order in which sprites are drawn, so the top sprite is always preferred. E..g., the Y-sorting here should really be using the sprite bounding box y-sorting used in `SpriteDrawingOrderComparer` (bounding box calculated in `ProcessSpriteEntities()`)